auction-deploy/auction_deploy/cli.py

```diff
@@ -78,6 +79,22 @@ class AuctionState(Enum):
 )
 
 
+auction_address_option = click.option(
+    "--auction-address",
+    help='The address of the auction contract, "0x" prefixed string',
+    type=str,
+    required=True,
+    callback=validate_address,
+    metavar="ADDRESS",
+)
+whitelist_file_option = click.option(
+    "--file",
```

Comment:
Please use: `"--file", "whitelist_file"` for readability

Reply:
I'll fix this in the simplify auction command PR